src/features/defi/providers/foxy/components/FoxyManager/Withdraw/components/Withdraw.tsx

```diff
@@ -183,7 +183,7 @@ export const Withdraw: React.FC<
           if (!estimatedGasCrypto) return
           dispatch({
             type: FoxyWithdrawActionType.SET_WITHDRAW,
-            payload: { estimatedGasCrypto },
+            payload: { estimatedGasCryptoBaseUnit: estimatedGasCrypto },
```

Comment:
We should rename the `const` on line `182` whilst we are here.

Reply:
613a7d8f64553ad701533511899ee87a44227a6d